README.md

```diff
@@ -76,7 +76,7 @@ kubectl label node $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}')
 
 Each workload resource may have additional constraints or scheduling affinities based on each node's labels in addition to the `bottlerocket.aws/updater-interface-version` label scheduling constraint.
 
-Customized deployments may use the [suggested deployment](./update-operator.yaml) or the [example development deployment](./dev/deployment.yaml) as a starting point, with customized container images specified if needed.
+Customized deployments may use the [suggested deployment](./update-operator.yaml) as a starting point, with customized container images specified if needed.
```

Comment:
was this change planned to be part of another commit where you are updating the read me?

Reply:
ah, missed that. i'll fix it